ouroboros-consensus/demo-playground/Run.hs

```diff
@@ -78,7 +87,6 @@ handleSimpleNode p CLI{..} (TopologyInfo myNodeId topologyFile) = do
 
     putStrLn $ "**************************************"
     putStrLn $ "I am Node = " <> show myNodeId
-    putStrLn $ "My consumers are " <> show (consumers nodeSetup)
```

Comment:
Because consumers will connect to us, when we start the node we don't know who will connect to us. `NodeSetup` does not contain `consumers` any more.

The information is not lost though, it is still in `NetworkTopology ~ [NodeSetup]` but one would need to go through `NodeSetup` of all the other nodes to see who will contact us; I just don't think this would be the right thing to do.